tensorflow_addons/layers/max_unpooling_2d.py

```diff
@@ -105,7 +107,13 @@ class MaxUnpooling2D(tf.keras.layers.Layer):
     """
 
     @typechecked
-    def __init__(self, pool_size=(2, 2), strides=(2, 2), padding="SAME", **kwargs):
+    def __init__(
+        self,
+        pool_size: Union[list, tuple, int] = (2, 2),
```

Comment:
Why do we need 'int' here?

Reply:
Pooling and conv layers also accept a single int to represent the same width and height, and `normalize_tuple` can deal with it.

Reply:
I see. Thanks.